tests/README.md

```diff
@@ -423,7 +423,7 @@ the test driver machine.
 
 * Start by making sure you're up to date, and install git and ducktape:
 
-        $ sudo apt-get update && sudo apt-get -y upgrade && sudo apt-get install -y git
+        $ sudo apt-get update && sudo apt-get -y upgrade && sudo apt-get install -y pip git
```

Comment:
So that will get you past pip, but I am stuck installing ducktape.

```
86_64-2.7/_openssl.c -o build/temp.linux-x86_64-2.7/build/temp.linux-x86_64-2.7/_openssl.o
    build/temp.linux-x86_64-2.7/_openssl.c:434:30: fatal error: openssl/opensslv.h: No such file or directory
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```
open ssl is installed and on the latest.